gtsfm/data_association/data_assoc.py

```diff
@@ -107,16 +109,22 @@ def run(
                 raise RuntimeError("Some cameras must have been dropped ")
             triangulated_data.add_camera(cam)
 
-        logger.debug(
-            "[Data association] output number of tracks: %s",
-            triangulated_data.number_tracks(),
-        )
-        logger.debug(
-            "[Data association] output avg. track length: %s",
-            SfmResult(triangulated_data, None).get_track_length_statistics()[0],
-        )
+        num_tracks = triangulated_data.number_tracks()
+        mean_track_length, median_track_length = SfmResult(triangulated_data, None).get_track_length_statistics()
+
+        logger.debug("[Data association] output number of tracks: %s", num_tracks)
+        logger.debug("[Data association] output avg. track length: %s", mean_track_length)
+
+        points_3d = [list(triangulated_data.track(j).point3()) for j in range(num_tracks)]
+        data_assoc_metrics = {
+            "num_tracks": num_tracks,
+            "mean_track_length": mean_track_length,
+            "median_track_length": median_track_length,
```

Comment:
I was just trying to dump as much as possible now, since space is cheap and performance is not great yet :-)

Do you mean median error within each track, or the median of all avg track errors @akshay-krishnan ?

Reply:
We would need to change the SFMResult class to get the min and max stats, so i'd prefer to leave that for another PR

Reply:
On second thought, i added that.

But "mean track length" is a metric I've seen in published papers like COLMAP so I'd like to keep it for now

Reply:
Then we probably won't need median. We'll ultimately end up using one number for the central tendency (bias) and one for the range/variance. But its upto you.